src/services/user/services/registrationConsent.js

```diff
@@ -0,0 +1,43 @@
+const { BadRequest } = require('@feathersjs/errors');
+
+
+class RegistrationConsentService {
```

Comment:
Can you please add, import and setup a empty hook chain?
Like in consent version Service.
To let other developers to know that this service do not include any hook at the moment and if any must add one in future that can easy pass it to the chain.

Reply:
done

Reply:
done